PWGHF/TableProducer/candidateCreatorB0.cxx

```diff
@@ -116,220 +115,146 @@ struct HfCandidateCreatorB0 {
     df3.setMaxDZIni(maxDZIni);
     df3.setMinParamChange(minParamChange);
     df3.setMinRelChi2Change(minRelChi2Change);
-    df3.setUseAbsDCA(useAbsDCA);
-    df3.setWeightedFinalPCA(useWeightedFinalPCA);
```

Comment:
I would let it with the configurables; in fact, it would be better to have the flexibility to use the same configuration used in the `track-index-skim-creator` for the D+ (while I agree to hard code it for the B0, since this is the only working configuration for vertices built with "pseudo" tracks)

Reply:
Indeed, thanks. I must have been working with a former version that did not take into account the PR of Fabio from 2 weeks ago.